articles/clustering-in-unsupervised-ml/index.md

```diff
@@ -0,0 +1,157 @@
+## Clustering in Unsupervised Machine Learning
+
+### Introduction to Unsupervised Machine Learning
+Unsupervised learning is a technique in machine learning (ML) that does not require the supervision of models by users. It is one of the categories of machine learning. The other two categories include reinforcement and supervised learning. 
+
+In unsupervised machine learning, a self-organized algorithm is utilized to enhance the self-discovery of unknown patterns that are in unlabeled datasets. 

```

Comment:
In unsupervised machine learning, a self-organized algorithm is used to enhance the self-discovery of unknown patterns in unlabeled datasets.

Reply:
Can you rephrase this into a simpler sentence? It doesn't sound effortless. Also, what is a self-organized algorithm? Is self in self-discovery necessary?

Reply:
I have made changes to simplify the sentence. I have also removed the self in self-discovery.